content/spin/v3/ai-sentiment-analysis-api-tutorial.md

```diff
@@ -42,6 +42,8 @@ In this tutorial we will:
 
 ## Tutorial Prerequisites
 
+> Please note that the code sample in this article is not using the latest version of the Spin SDK, but works with the latest version of Spin (3.0.0). Please refer to the code in this [GitHub repository for the working samples and corresponding SDK version](https://github.com/fermyon/ai-examples/tree/main).
```

Comment:
I do not think we made any breaking changes regarding the SDK, and things should work, which confuses me.

Reply:
I tried with the Rust sample and I had to change it to make it work.

This is the diff I ended up with to make the Rust sample work:
`5c5
<     http::{IntoResponse, Params, Request, Response, Router},
---
>     http::{IntoResponse, Json, Params, Request, Response, Router},
56c56,59
< fn perform_sentiment_analysis(req: Request, _params: Params) -> Result<impl IntoResponse> {
---
> fn perform_sentiment_analysis(
>     req: http::Request<Json<SentimentAnalysisRequest>>,
>     _params: Params,
> ) -> Result<impl IntoResponse> {
58,60c61
<     let sentiment_analysis_request: SentimentAnalysisRequest =
<         serde_json::from_slice(req.body()).expect("Failed to deserialize request body");
<     let sentence = sentiment_analysis_request.sentence.trim();
---
>     let sentence = req.body().sentence.trim();`

Reply:
Error with the orginal code 
```
├╴E failed to resolve: use of undeclared crate or module `http`
 │   use of undeclared crate or module `http` rustc (E0433) [57, 10]
`
```

Never templates don't have 'http' as a dependency - can that be it?

Reply:
The sample in the repo, which works without any changes use a fairly old SDK `1.5.0`. I haven't checked if the other languages work based on new SDK version and "old" code sample.